torch/_refs/__init__.py

```diff
@@ -2779,6 +2781,70 @@ def _normalize(
     return out, mean, rstd
 
 
+# remove all specified dimensions
+def _squeeze_multiple(x: TensorLikeType, dimensions: List[int]) -> TensorLikeType:
```

Comment:
I think the dimensions can be canonicalized by the caller of this helper and instead of this `prims.squeeze` can be used directly? 

https://github.com/pytorch/pytorch/blob/dd43903fa99b8549225ec63c2e81ef4693436be0/torch/_prims/__init__.py#L1702